### PR TITLE
Embedding UX

### LINKED ISSUE #6375: revamp embeddings grouping demo page UI
Overview: 
- Single Input to  add the error (body)
- Introduction as a description + link to showhn/docs
- After input, List of Tags & similiar errors.
- Not part of app -> No Navigation etc.

Image as reference: 
image

### PR DESCRIPTION
**Pull request by deltaepsilon**
## Summary

closes #6375 

The path is `app.highlight.io/error-tags`

image

image

### CODE REVIEW

**Comment by deltaepsilon** [comment]
@julian-highlight What do you think of these screenshots?

File: frontend/src/routers/AppRouter/AppRouter.tsx [hunk 1]
```diff
@@ -211,6 +217,21 @@ export const AppRouter = () => {
 				<WithSessionSearchContext>
 					<WithErrorSearchContext>
 						<Routes>
+							<Route
+								path="/error-tags"
+								element={<ErrorTagsContainer />}
+							>
+								<Route index element={<ErrorTags />} />
+								<Route
+									path="/error-tags/admin"
```
Comment (Vadman97) [inline frontend/src/routers/AppRouter/AppRouter.tsx:226]: should this route be only accessible to us (highlight team)?

File: sdk/firstload/package.json [hunk 2]
```diff
@@ -28,7 +28,7 @@
 		"build": "yarn typegen && vite build && yarn build:umd",
 		"build:umd": "cp dist/index.umd.cjs dist/index.umd.js",
 		"build:watch": "yarn build --watch",
-		"dev": "vite dev",
+		"dev": "",
```
Comment (Vadman97) [inline sdk/firstload/package.json:31]: why this change?

**Review by Vadman97 — COMMENTED** [review]
looks like build is failing due to imports
also some questions on the design / UX for you and @julian-highlight 
https://www.loom.com/share/bf72791a7569474493728253bba8dc18

**Comment by julian-highlight** [comment]
> 

do we need this step in-between? As far as I remember we wanted to simply have the first page with a single input?
If loading times are as long as in the PR we should defo add a loading state. Let me know if the loading times are expected like presented in the loom! 

Also we came to the conclusion on Slack that this UI is nicer for the first page! Sorry if that went missing somewhere, but since then the designs have been updated. 
https://www.figma.com/file/rdBPNzn7Klk6RG1LHUCE5B/Screen-Designs?type=design&node-id=11353-340426&mode=design&t=Tgs9pS5JEYlZ140T-11

CC @deltaepsilon

File: frontend/src/routers/AppRouter/AppRouter.tsx (hunk 1 above)
Comment (deltaepsilon) [reply frontend/src/routers/AppRouter/AppRouter.tsx:226]: Yeah. It should. Do we have an easy way to do this?

**Comment by deltaepsilon** [comment]
> looks like build is failing due to imports also some questions on the design / UX for you and @julian-highlight https://www.loom.com/share/bf72791a7569474493728253bba8dc18

@Vadman97 I'm confused. It looks like you're reviewing the stubbed-in version.

I developed the full thing like the screenshots at the top of this page. Those are screenshots of the live application on my machine, not the mocks. I already implemented the designs that @julian-highlight is referring to.

File: sdk/firstload/package.json (hunk 2 above)
Comment (deltaepsilon) [reply sdk/firstload/package.json:31]: I shouldn't have checked that in. 

We can't run both `yarn dev:frontend` and `yarn e2e:nextjs` at the same time, because this `vite dev` job fights over a port. I've been manually disabling the job to get both running. The better solution is probably to not run `dev` for `firstload` when running `yarn e2e:nextjs`.

**Comment by deltaepsilon** [comment]
https://github.com/highlight/highlight/assets/878947/ed26947d-c0ab-415c-808b-3ea87fb14857

I'm committing my last fixes. This is the current state of the feature 👆

File: frontend/src/routers/AppRouter/AppRouter.tsx (hunk 1 above)
Comment (deltaepsilon) [reply frontend/src/routers/AppRouter/AppRouter.tsx:226]: Fixed!

File: sdk/firstload/package.json (hunk 2 above)
Comment (deltaepsilon) [reply sdk/firstload/package.json:31]: Fixed!

**Comment by julian-highlight** [comment]
> error-tags.mp4 
> I'm committing my last fixes. This is the current state of the feature 👆

Love the tunes! Looks great to me! I'd prefer the updated version for the first screen, but this already works pretty sweet! I'll do a quick design for a loading screen, just to give some context to users that there's something loading!

**Comment by julian-highlight** [comment]
There's the loading state: https://www.figma.com/file/rdBPNzn7Klk6RG1LHUCE5B/Screen-Designs?type=design&node-id=11388-447608&mode=design&t=Tgs9pS5JEYlZ140T-11

If you come up with even better copy feel free to change! @deltaepsilon

**Review by Vadman97 — APPROVED** [review]
my bad here, i was looking at a render.com preview that was outdated (it didn't build the latest commit for some reason).
my only comment is that the results are overflowing, but this looks good otherwise!
Screenshot 2023-08-31 at 3 39 42 PM
